includes/helm.py

```diff
@@ -241,74 +241,33 @@ def get_info_from_helm(component, repo, services):
-        existing_alertmanager_config = {
-          'alert_severity_label': get_existing_env_config(
-            component, env, 'alert_severity_label', services
-          ),
-          'alerts_slack_channel': get_existing_env_config(
-            component, env, 'alerts_slack_channel', services
-          ),
-        }
-        alertmanager_config = {
-          'alert_severity_label': None,
-          'alerts_slack_channel': None,
-        }
-
-        if generic_prometheus_alerts := values.get('generic-prometheus-alerts'):
-          log.debug(
-            f'generic-prometheus alerts found in values: {generic_prometheus_alerts}'
-          )
-          if alert_severity_label := generic_prometheus_alerts.get('alertSeverity'):
-            log.debug(f'updating {env} alert_severity_label to {alert_severity_label}')
-            alertmanager_config['alert_severity_label'] = alert_severity_label
-            # Only look for the slack channel if the alert_severity_label tehre
-            if alerts_slack_channel := am.find_channel_by_severity_label(
-              alert_severity_label
-            ):
-              alertmanager_config['alerts_slack_channel'] = alerts_slack_channel
-              log.debug(
-                f'updating {env} alerts_slack_channel to {alerts_slack_channel}'
-              )
-            else:
-              log.warning(
-                f'Alerts slack channel not found for {alert_severity_label} in {env} values file'
-              )
-        # fallback to a default severity label
-        if (
-          not alertmanager_config['alert_severity_label']
-          and alert_severity_label_default
-        ):
-          log.info(
-            f'Alert severity label not found for {component_name} in {env} - attempting to set default'
-          )
-          alertmanager_config['alert_severity_label'] = alert_severity_label_default
-          if alerts_slack_channel := am.find_channel_by_severity_label(
-            alert_severity_label_default
-          ):
+        if am.isDataAvailable():
+          # Alert severity label and slack channel
+          alertmanager_config = {
+            'alert_severity_label': None,
+            'alerts_slack_channel': None,
+          }
+         
+          if generic_prometheus_alerts := values.get('generic-prometheus-alerts'):
+            if alert_severity_label := generic_prometheus_alerts['alertSeverity']:
+              log.debug(f'generic-prometheus alerts found in values: {generic_prometheus_alerts}')
+              log.debug(f'Updating {env} alert_severity_label to {alert_severity_label}')
+              alertmanager_config['alert_severity_label'] = alert_severity_label
+          if alert_severity_label_default and not alertmanager_config['alert_severity_label']:
+            log.info(f'Alert severity label not found for {component_name} in {env} - setting to default')
+            alertmanager_config['alert_severity_label'] = alert_severity_label_default
```

Comment:
It looks like we're not setting `alert_severity_label` in this `if` but need it on line 262?